### PR TITLE
Release 0.8.1

### PR DESCRIPTION
**Pull request by shanshin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+0.8.1 / 2024-06-07
+===================
+## Kover Gradle Plugin
+
+### Features
+* [`#600`](https://github.com/Kotlin/kotlinx-kover/issues/600) Added DSL to copy one report variant
+* Added DSL to copy one report variant
```
Comment (sandwwraith) [inline CHANGELOG.md:7]: This is a duplication of the previous point. Perhaps you wanted to mention DSL enhancement?

File: build-logic/src/main/kotlin/kotlinx/kover/conventions/kover-release-conventions.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2017-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+/*
+ * Copyright 2000-2024 JetBrains s.r.o.
```
Comment (sandwwraith) [inline build-logic/src/main/kotlin/kotlinx/kover/conventions/kover-release-conventions.gradle.kts:9]: Copyright duplicated

File: build-logic/src/main/kotlin/kotlinx/kover/conventions/kover-release-conventions.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        } else {
+            value
+        }
+    }.joinToString(".")
+
+    return "$newVersion-SNAPSHOT"
+}
+
+fun File.replaceInFile(old: String, new: String) {
+    if (name.endsWith(".png")) return
```
Comment (sandwwraith) [inline build-logic/src/main/kotlin/kotlinx/kover/conventions/kover-release-conventions.gradle.kts:120]: we have PNGs?

File: kover-cli/build.gradle.kts [hunk 4]
```diff
@@ -21,6 +21,7 @@ plugins {
     id("kover-publishing-conventions")
     id("kover-docs-conventions")
     id("kover-fat-jar-conventions")
+    id("kover-release-conventions")
```
Comment (sandwwraith) [inline kover-cli/build.gradle.kts:24]: Why this plugin have to be applied in sub-projects?

File: build-logic/src/main/kotlin/kotlinx/kover/conventions/kover-release-conventions.gradle.kts (hunk 3 above)
Comment (shanshin) [reply build-logic/src/main/kotlin/kotlinx/kover/conventions/kover-release-conventions.gradle.kts:120]: Yep, `kover-gradle-plugin/docs/html.png`

File: kover-cli/build.gradle.kts (hunk 4 above)
Comment (shanshin) [reply kover-cli/build.gradle.kts:24]: Because each subproject has its own directory of `docs` and `examples`, and I want to keep the project isolation

**Review by sandwwraith — APPROVED** [review]

File: kover-gradle-plugin/docs/migrations/migration-to-0.8.0.md [hunk 5]
```diff
@@ -124,7 +124,7 @@ If you use the extensions property to access the extension
 The `0.7.x` version used two project extensions named `kover` and `koverReports`.
 This was confusing and made it difficult to search through documentation or examples.
 
-Since `0.8.0`, there is only one `kover` extension left.
+Since `0.8.1`, there is only one `kover` extension left.
```
Comment (MGaetan89) [inline kover-gradle-plugin/docs/migrations/migration-to-0.8.0.md:127]: Are these changes meant to be here? Shouldn't it remain `0.8.0`?
They happened multiple times in this file

Comment (shanshin) [reply kover-gradle-plugin/docs/migrations/migration-to-0.8.0.md:127]: In this case, yes, it is worth leaving `0.8.0`.

For this case, we can rephrase it like: in any version `0.8.x` there is only one `kover` extension left.

Comment (shanshin) [reply kover-gradle-plugin/docs/migrations/migration-to-0.8.0.md:127]: Fixed, thanks!

**Pull request merged**